extensions/mssql/src/objectManagement/ui/databaseDialog.ts

```diff
@@ -100,24 +100,39 @@ export class DatabaseDialog extends ObjectManagementDialogBase<Database, Databas
 			this.initializeStateSection();
 
 			// Initilaize general Tab
+			const generalTabContent = this.createGroup('', [
+				this.databaseSection,
+				this.backupSection
+			], false);
 			this.generalTab = {
 				title: localizedConstants.GeneralSectionHeader,
 				id: this.generalTabId,
-				content: this.createGroup('', [
-					this.databaseSection,
-					this.backupSection
-				], false)
+				content: generalTabContent,
+				dispose: () => {
+					generalTabContent.dispose();
+				}
 			};
 
 			// Initilaize Options Tab
+			const optionTabContent = this.createGroup('', this.optionsTabSectionsContainer, false);
 			this.optionsTab = {
 				title: localizedConstants.OptionsSectionHeader,
 				id: this.optionsTabId,
-				content: this.createGroup('', this.optionsTabSectionsContainer, false)
+				content: optionTabContent,
+				dispose: () => {
+					optionTabContent.dispose();
+				}
 			};
 
 			// Initilaize tab group with tabbed panel
-			const propertiesTabGroup = { title: '', tabs: [this.generalTab, this.optionsTab] };
+			const propertiesTabGroup = <azdata.TabGroup>{
+				title: '',
+				tabs: [this.generalTab, this.optionsTab],
+				dispose: () => {
+					this.generalTab.dispose();
```

Comment:
Same here, disposing a TabGroup should automatically dispose the tabs contained within it.